pkg/apiserver/controller/queue/queue.go

```diff
@@ -419,6 +426,32 @@ func UpdateQueue(ctx *logger.RequestContext, request *UpdateQueueRequest) (Updat
 	return response, nil
 }
 
+func validateQueueResource(resource schema.ResourceInfo, queueResource *schema.ResourceInfo) error {
+	scalarResourceLaws := config.GlobalServerConfig.Job.ScalarResourceArray
+	if resource.CPU != "" {
+		queueResource.CPU = resource.CPU
+	}
+	if resource.Mem != "" {
+		queueResource.Mem = resource.Mem
+	}
+	if resource.ScalarResources != nil {
+		queueResource.ScalarResources = make(schema.ScalarResourcesType)
+	}
+	if len(resource.ScalarResources) != 0 || resource.ScalarResources == nil {
```

Comment:
When `resource.ScalarResources == nil` ,  `range rsource.ScalarResources` might be panic.

Reply:
range nil resource.ScalarResources won't effect